extensions/security/runtime/src/main/java/io/quarkus/security/runtime/interceptor/PermitAllInterceptor.java

```diff
@@ -0,0 +1,27 @@
+package io.quarkus.security.runtime.interceptor;
+
+import javax.annotation.Priority;
+import javax.annotation.security.PermitAll;
+import javax.inject.Inject;
+import javax.interceptor.AroundInvoke;
+import javax.interceptor.Interceptor;
+import javax.interceptor.InvocationContext;
+
+/**
+ * @author Michal Szynkiewicz, michal.l.szynkiewicz@gmail.com
+ *         <br>
+ *         Date: 05/10/2019
+ */
+@Interceptor
+@PermitAll
+@Priority(1)
```

Comment:
Instead of magic numbers, you should probably use a combination of a predefined constant plus/minus a number (applies to all interceptors added here).
See the values here - https://javaee.github.io/javaee-spec/javadocs/javax/interceptor/Interceptor.Priority.html
I am not sure what would be appropriate for security interceptors though; I guess you want them to trigger very soon.

Reply:
`LIBRARY_BEFORE` seems most appropriate, wdyt?